### DISCUSSION TITLE
Installing ComfyUI Custom Node Dependencies on Google Drive with Colab

### DISCUSSION THREAD

**Discussion by Amit30swgoh** [Comfy-Org/ComfyUI-Manager#588] [General]
Hi everyone,

I'm using Google Colab with Cloudflare and have my Google Drive mounted. My goal is to install all the dependencies for my custom ComfyUI nodes directly within my ComfyUI folder on Google Drive. Here's what I'm hoping to achieve:

Avoid Repetitive Installs: I want to prevent the same dependencies from uninstalling and then reinstalling at the same version each time I mount my drive.
Error Prevention: I want to eliminate errors like missing 'torch', 'einops', etc. when mounting my drive and running Cloudflare setup.
Has anyone successfully set up something similar? Any tips or insights would be greatly appreciated!  I'd like to have a setup where the only thing I need to run is 'main.py' after mounting my drive.

Thanks in advance!

**Comment by ltdrdata** [comment]
This will soon be implemented through the restore-dependencies of `cm-cli`.

**Reply by Amit30swgoh** [reply]
You are the best TY DR!

**Reply by ltdrdata** [reply]
Notebook is updated.
